test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs

```diff
@@ -93,68 +93,344 @@ await foreach (Page<Product> page in allPages.AsPages())
         });
 
         [Test]
-        public Task RawLRO() => Test(async (host) =>
+        [TestCase(true)]
+        [TestCase(false)]
+        public Task RawLRO(bool nestedScopeSuppression) => Test(async (host) =>
         {
             using var diagnosticListener = new ClientDiagnosticListener("dpg_customization_LowLevel", asyncLocal: true);
             CollectionAssert.IsEmpty(diagnosticListener.Scopes);
 
-            Operation<BinaryData> result = await new DPGClient(Key, host, null).LroAsync(WaitUntil.Started, "raw");
+            Operation<BinaryData> result = await new DPGClient(Key, host, null, nestedScopeSuppression).LroAsync(WaitUntil.Started, "raw");
             diagnosticListener.AssertAndRemoveScope("DPGClient.Lro");
             CollectionAssert.IsEmpty(diagnosticListener.Scopes);
 
             await result.WaitForCompletionAsync();
             diagnosticListener.AssertAndRemoveScope("DPGClient.Lro.WaitForCompletion");
-            CollectionAssert.IsEmpty(diagnosticListener.Scopes);
+            if (nestedScopeSuppression)
```

Comment:
I think no matter `nestedScopeSuppression` is true or not, we should both check `CollectionAssert.IsEmpty(diagnosticListener.Scopes);` ?